docs/_posts/2019-02-27-test-post-featured-images.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Adding featured images to blog posts"
```

Comment:
this should be removed before the PR is merged

Reply:
i removed this test post, and replaced with Sigal's post.